### ISSUE TITLE
Allow more mobs to be considered valid sacrifices

### ISSUE THREAD

**Issue by cybercat5555** [Um-Mitternacht/Bewitchment#272]
Since there has been an influx of people asking about MCA villagers in the demon ritual, this issue is just to make it a publicly known issue. 
The following mobs should all be considered valid offerings in the demon ritual, and any similar ones:

- Vanilla Villagers (already in)

- MCA villagers, as people have tried to use them and it did not work

- Another player.  I could see issues with this, such as if the player starts the ritual with no other players nearby, it could instead kill them (which does make sense though), as well as issues if the targeted player moves out of the detection radius. To resolve this, the ritual should start as soon as the offering is killed in the circle, be it a vanilla villager, a MCA villager, or another player, instead of being activated with the foci glyph which then auto-kills the target, requiring the summoner to get more physical, adds to the theme tbh.

So in short in regards to the last bullet: ritual does not start until the target is killed manually while in the circle, and the required reagents are already in the circle. After all the proper ingredients are in and then the sacrifice killed, the ritual starts automatically. R-clicking on the foci for the sacrifice rituals no longer activates it, but does give a message stating "you must sacrifice the valid offering first"

**Comment by DaveArndt** [comment]
Should all rituals that require sacrifice have the same logic, or are we just doing this for demon summoning?

**Comment by cybercat5555** [comment]
I'd say all ones, also so we can prepare for future rituals that may require sacrifice. I'd probably not bother with the imps though, the chicken offering seems fine.

**Comment by Sunconure11** [comment]
All rituals that have sacrifices should use the same logic. The logic is a bit odd anyways,

**Comment by Sunconure11** [comment]
Due to changes in the system, more mobs are capable of being sacrifices now
